packages/core/src/Executor.ts

```diff
@@ -136,6 +136,7 @@ export class Executor {
       if(hook) return hook({
         isAvailable: () => this.memory.getNodeStatus(node.id) === 'AVAILABLE',
         input: this.memory.getInputDevice(node.id)!,
+        // todo: It seems the params didn't evaluate correctly
```

Comment:
Is this still an issue? If so can you expand on how to recreate it? 🙏

Reply:
I noticed that the `Await Node` utilizes the `canRun` hook, which accepts input and parameters. Within `canRun`, it assumes that the passed `params.value` has been formatted, however, the `value` has not been formatted.

```Await.ts
  canRun({ input, params }) {
    const haveChunk = input.haveItemsAtInput(
      'input',
       // Here, it expects to receive a number, but the data obtained has not been evaluated.
      (params.number_of_items.value as StringableInputValue).value as number
    )

    const haveRemainder = input.haveAllItemsAtInput('input')

    return haveChunk || haveRemainder
  },
  ```